qiling/debugger/qdb/utils.py

```diff
@@ -1,27 +1,15 @@
 #!/usr/bin/env python3
-from qiling.const import *
+
 from functools import partial
 
+from qiling.const import *
 
+PROGRAM_EXITED = "DONE"
```

Comment:
DONE ? why not true or false ?

Reply:
this value will be returned as a address, so true or false will be converted to 0x1 or 0x0 by python automatically, will mess up the breakpoint thing , but sure i can change it